### PR TITLE
mvebu: misc cleanup of board definitions

### PR DESCRIPTION
**Pull request by rmounce**

### CODE REVIEW

File: target/linux/mvebu/base-files/lib/upgrade/platform.sh [hunk 1]
```diff
@@ -16,10 +16,7 @@ platform_do_upgrade() {
 	armada-385-linksys-caiman|armada-385-linksys-cobra|armada-385-linksys-rango|armada-385-linksys-shelby|armada-xp-linksys-mamba)
 		platform_do_upgrade_linksys "$ARGV"
 		;;
-	armada-388-clearfog-pro)
-		platform_do_upgrade_clearfog "$ARGV"
-		;;
-	armada-388-clearfog-base)
+	armada-388-clearfog|armada-388-clearfog-base|armada-388-clearfog-pro)
```
Comment (KanjiMonster) [inline target/linux/mvebu/base-files/lib/upgrade/platform.sh]: There is no "armada-388-clearfog" anymore, it has been split into armada-388-clearfog-base and armada-388-clearfog-pro. So either drop armada-388-clearfog or use armada-388-clearfog-* as 02_network does.

File: target/linux/mvebu/base-files/lib/upgrade/platform.sh [hunk 2]
```diff
@@ -32,7 +29,7 @@ platform_copy_config() {
 	armada-385-linksys-caiman|armada-385-linksys-cobra|armada-385-linksys-rango|armada-385-linksys-shelby|armada-xp-linksys-mamba)
 		platform_copy_config_linksys
 		;;
-	armada-388-clearfog)
+	armada-388-clearfog|armada-388-clearfog-base|armada-388-clearfog-pro)
```
Comment (KanjiMonster) [inline target/linux/mvebu/base-files/lib/upgrade/platform.sh]: Same here.

**Comment by rmounce** [comment]
Amended, thanks

**Comment by rmounce** [comment]
Already fixed those bits :)

**Comment by KanjiMonster** [comment]
> Already fixed those bits :)

You are too fast ;p

**Comment by hauke** [comment]
Thank you for the patch, it was applied to master.

**Pull request closed without merging**